src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -116,9 +120,10 @@ private void generateReport(String inputDates) throws Exception {
         RepoConfiguration.setDatesToRepoConfigs(
                 repoConfigs, cliArguments.getSinceDate(), cliArguments.getUntilDate());
 
-        ReportGenerator.generateReposReport(repoConfigs, FT_TEMP_DIR, TEST_REPORT_GENERATED_TIME,
-                cliArguments.getSinceDate(), cliArguments.getUntilDate(), cliArguments.isSinceDateProvided(),
-                cliArguments.isUntilDateProvided(), () -> TEST_REPORT_GENERATION_TIME);
+        ReportGenerator.generateReposReport(repoConfigs, FT_TEMP_DIR, FT_ASSETS_TEMP_DIR, new ReportConfiguration(),
+                TEST_REPORT_GENERATED_TIME, cliArguments.getSinceDate(), cliArguments.getUntilDate(),
+                cliArguments.isSinceDateProvided(), cliArguments.isUntilDateProvided(), () ->
```

Comment:
It was a trick that I'm using to avoid checkstyle errors for now. Maybe we can update checkstyle configurations in the future to account for this scenario?